src/middlewares/withLogger.ts

```diff
@@ -0,0 +1,31 @@
+import { createLogger, EncoderTypeEnum } from "@fwl/logging";
+import { ServerResponse } from "http";
+
+import { Handler } from "../@types/ApiPageHandler";
+import { ExtendedRequest } from "../@types/ExtendedRequest";
+
+const logger = createLogger({
+  service: "connect-account",
+  encoder: EncoderTypeEnum.JSON,
+}).withMeta({
+  process: "apiPage",
```

Comment:
This could also be SSR, right?

Reply:
this is

Reply:
So we don't differentiate API routes from getServerSidesProps functions?